app/views/spotlight/shared/_curation_sidebar.html.erb

```diff
@@ -17,4 +17,7 @@
   <% if (can? :manage, current_exhibit.translations.first_or_initialize) && current_exhibit.languages.any? %>
     <%= nav_link t(:'spotlight.curation.sidebar.translations'), spotlight.edit_exhibit_translations_path(current_exhibit), 'data-no-turbolink' => true %>
   <% end %>
+  <% if can? :curate, current_exhibit %>
```

Comment:
I wonder if we want to use a different ability to let downstream consumers limit who can bulk update separate from other curator actions? (And, we could limit it to superadmins in exhibits until we've got all the pieces in place?)

Reply:
That was certainly brought up in our initial spike, but given that bulk actions only check if the user could curate the exhibit we followed the same pattern.  I'm happy to go down that route though.  If we do, a few questions come to mind.

What should that ability be?  Should we also be checking the same/similar ability for bulk actions?

Reply:
Went with `bulk_updates` and only am checking it for the bulk update admin UI.